Tests/OptimizelyTests-DataModel/ExperimentTests.swift

```diff
@@ -192,6 +196,8 @@ extension ExperimentTests {
             "\"us\" OR \"female\"",
             "(\"us\" OR \"female\") AND \"adult\"",
             "(\"us\" OR (\"female\" AND \"adult\")) AND (\"fr\" AND (\"male\" OR \"kid\"))",
+            "NOT (\"us\" AND \"female\")",
```

Comment:
Use `#"raw "string" which is in quotes"#`?

Reply:
I didn't know about this trick :) I'll skip it in this case.

Reply:
Swift 5+, no big deal!